spec/jasmine-helper.js

```diff
@@ -0,0 +1,222 @@
+'use babel'
+/* @flow */
```

Comment:
@maxbrunsfeld Actually, I just asked and apparently `// @flow` should also work today. I'm still following up on a `.flowconfig` option, though.